testing/travis-fork-check.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import os
+import subprocess
+import sys
+
+def check_travis_repo_slug():
+    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
+        sys.stderr.write("Decryption disabled. Setting MANTL_CI_FORK_CHECK to 1")
+        print("1")
+    else:
+        sys.stderr.write("Decrypting OS ssh key")
+        cmd = ["openssl",
```

Comment:
why is this script still decrypting the key? Isn't that happening in line 38 of travis.yml now?